pr_insight/tools/pr_code_suggestions.py

```
@@ -4,6 +4,7 @@
 from functools import partial
 from typing import Dict, List
 from jinja2 import Environment, StrictUndefined
+from bs4 import BeautifulSoup, Comment
```

Ruff

`bs4.BeautifulSoup` imported but unused

(F401)